frontend/public/kubevirt/components/cluster/cluster-overview.jsx

```diff
@@ -147,6 +152,15 @@ export class ClusterOverview extends React.Component {
     }));
   }
 
+  _setCephDiskStats(key, response) {
+    this.setState(state => ({
```

Comment:
there many PRs and I mentioned it only somewhere, but these methods should be made more generic